installer/templates/new/web/templates/layout/app.html.eex

```diff
@@ -27,6 +27,8 @@
 
     </div> <!-- /container -->
     <script src="<%%= static_path(@conn, "/js/app.js") %>"></script>
-    <script>require("web/static/js/app")</script>
+    <%= if brunch do %>
+      <script>require("web/static/js/app")</script>
```

Comment:
Let's not indent here and let's make it so there is no extra white space with the eex `if`